app/views/layouts/application.haml

```diff
@@ -111,6 +111,10 @@
             - if logged_in?
               %li= link_to 'Unread', unread_posts_path
               %li= link_to 'Replies Owed', owed_posts_path
+              - if current_user.replies_owed_indicator
+                - $owed = get_owed
+                - if $owed
```

Comment:
You should probably check > 0 here; despite also being C-based, Ruby (unlike Python) generally treats 0 as truthy.